params/es_params.go

```diff
@@ -1,8 +1,9 @@
 package params
 
 const (
-	EsGetBlobPerByteGas = 3 // Per-byte price for an esGetBlob operation.
-	EsGetBlobBaseGas = 50000 // Base price for an esGetBlob operation.  
+	EsGetBlobPerByteGas uint64 = 1 // Per-byte price for an esGetBlob operation.
+	EsGetBlobBaseGas    uint64 = 50000 // Base price for an esGetBlob operation.
+	ESMemoryGas         uint64 = 1
```

Comment:
I believe our current w3q testnet uses 3.  Given the block gas limit is 100M, the maximum memory size can be 100M * 32 = 3.2GB - it is possible to introduce new attacks?